packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -35,6 +35,11 @@ const Color _kSelectionHighlightColor = Color(0x667FAACF);
 const Color _kInactiveTextColor = Color(0xFFC2C2C2);
 const Color _kDisabledBackground = Color(0xFFFAFAFA);
 
+// An eyeballed value that moves the cursor slightly left of where it is
+// rendered by default so its positioning more accurately matches the native
+// iOS cursor positioning.
+const int _iOSHorizontalCursorOffsetPixels = -2;
```

Comment:
Since we conventionally use "pixels" to refer to logical pixels in most of the codebase, specify that it's physical pixels either in the name or in the comments.